shared/src/commonMain/kotlin/org/hyperskill/app/study_plan/widget/presentation/StudyPlanWidgetReducer.kt

```diff
@@ -108,8 +118,8 @@ class StudyPlanWidgetReducer : StateReducer<State, Message, Action> {
         if (state.sectionsStatus == StudyPlanWidgetFeature.ContentStatus.IDLE ||
             state.sectionsStatus == StudyPlanWidgetFeature.ContentStatus.ERROR && message.forceUpdate
         ) {
-            State(sectionsStatus = StudyPlanWidgetFeature.ContentStatus.LOADING) to
-                setOf(InternalAction.FetchStudyPlan())
+            state.copy(sectionsStatus = StudyPlanWidgetFeature.ContentStatus.LOADING) to
```

Comment:
`Message.Initialize` and its handling function `coldContentFetch` could be invoked on:

1. Initial data loading (feature initialization)
2. When retry content loading button clicked

so it's better to reset State (create a new one with default values), like before - we are starting data loading from scratch

```suggestion
            State(sectionsStatus = StudyPlanWidgetFeature.ContentStatus.LOADING) to
```